README.md

```diff
@@ -68,6 +68,7 @@ For example, if a package supports the css file extraction you can run the autop
 | [smart-css](https://github.com/hackhat/smart-css) | 1.1.1 | | ✓ | ✓ | ✓ | |
 | [stile](https://github.com/bloodyowl/stile) + [react-media-queries](https://github.com/bloodyowl/react-media-queries) | 2.0.0 | ✓ | | ✓ | ✓ | | |
 | [stilr](https://github.com/kodyl/stilr) | 1.1.0 | | ✓ | ✓ | ✓ | ✓ |
+| [styable](https://github.com/wix/stylable) | 5.0.1 | | ✓ | ✓ | N/A (CSS syntax) | N/A (transpiles to flat CSS) |
```

Comment:
`N/A (CSS syntax)` -  from my POV this answers as **no** to "Styles As Object Literals". Leave it blank
` N/A (transpiles to flat CSS)` - this answers as **yes** to "Extract CSS File". Use tick here. Or I miss something here?